tests/unit/test-class-wc-payment-gateway-wcpay-payment-types.php

```diff
@@ -131,6 +147,7 @@ public function set_up() {
 					$this->mock_rate_limiter,
 					$this->mock_order_service,
 					$mock_dpps,
+					$this->mock_localization_service,
```

Comment:
Here we don't need a new property, and we can dispatch `$this->createMock( WC_Payments_Localization_Service::class )` right here.

Reply:
Done https://github.com/Automattic/woocommerce-payments/pull/6920/commits/3dc1eb4d18404c33265e1217b9157f87c679920e